inst/htmlwidgets/lib/justgage/justgage.css

```diff
@@ -1,15 +1,9 @@
-
-
 .html-widget.gauge {
   height: 160px;
 }
 
 .html-widget.gauge svg {
   height: 160px;
   margin-top: -10px;
-  margin-bottom: -40px;
+  margin-bottom: -30px;
```

Comment:
Pretty sure this is needed since the title feature was dropped when upgrading in #301